plugins/hls-pragmas-plugin/src/Ide/Plugin/Pragmas.hs

```diff
@@ -163,9 +166,19 @@ completion _ide _ complParams = do
                     | "{-# LANGUAGE" `T.isPrefixOf` VFS.fullLine pfix
                     = J.List $ map buildCompletion
                         (Fuzzy.simpleFilter (VFS.prefixText pfix) allPragmas)
+                    | "{-# options_ghc" `T.isPrefixOf` T.toLower (VFS.fullLine pfix)
+                    = J.List $ map mkExtCompl
+                        (Fuzzy.simpleFilter (VFS.prefixText pfix) flags)
```

Comment:
do we want to add the closing bracket (when needed) here as well?

Reply:
actually no, this is the case that fills in the values of the options, not the name. The completion for the string "{-# options_ghc ... #-} is handled in the other case and handles closing brackets.